virtcontainers/monitor.go

```diff
@@ -37,7 +37,7 @@ func (m *monitor) newWatcher() (chan error, error) {
 	m.Lock()
 	defer m.Unlock()
 
-	watcher := make(chan error, 1)
+	watcher := make(chan error, 128)
```

Comment:
Suggestion (but not required since this is simply changing what's already there): We could define this as a `const` at the top-level of the file with an explanatory comment.